doc/using-the-pandoc-api.md

```diff
@@ -313,10 +313,10 @@ Note that this looks first in the
 users to override the system defaults.  If you want to disable
 this behavior, use `setUserDataDir Nothing`.
 
-To render a template, use `renderTemplate'`, which takes two
```

Comment:
ACtually it is `renderTemplate'`, the prime is there in the function name.
(There's also `renderTemplate`, which is different.)